webapp/src/components/Description.test.tsx

```diff
@@ -2,24 +2,26 @@ import { screen } from "@testing-library/react";
 import Description from "components/Description";
 import { renderWithTheme } from "mocks/utils";
 
-test("display description with just text", async () => {
-  renderWithTheme(<Description text="test description" />);
+describe("Description", () => {
+  test("display description with just text", async () => {
```

Comment:
That's just a detail, but if you switch to the `describe()` pattern, it usually goes well with the `it("should ...` pattern.
```suggestion
  it("should display description with text only", async () => {
```